test/run-js-linter.sh

```diff
@@ -10,6 +10,3 @@ eslint jsx/
 
 # Run ESLint on all JS files (see exceptions in .eslintignore)
 eslint htdocs/js/
-
-# Other files
-eslint Gruntfile.js
```

Comment:
Would be nice to add lint for `webpack.config.js` instead of removed `Gruntfile.js`